apps/fp-frontend-app/src/data/behandlingApi.ts

```diff
@@ -225,18 +224,10 @@ const getKlageVurderingOptions = (links: ApiLink[]) => (behandling: Behandling)
     staleTime: Infinity,
   });
 
-//TODO (TOR) Denne lenka får med uuid fra backend, men treng saksnummer. Sett heller saksnummer backend så slepp ein det her
-const getInnsynDokumenterOptions = (links: ApiLink[]) => (fagsak: Fagsak, behandling: Behandling) =>
+const getInnsynDokumenterOptions = (links: ApiLink[]) => (behandling: Behandling) =>
   queryOptions({
-    queryKey: [BehandlingRel.INNSYN_DOKUMENTER, fagsak.saksnummer, behandling.uuid, behandling.versjon],
-    queryFn: () =>
-      kyExtended
-        .get(getUrlFromRel('INNSYN_DOKUMENTER', links), {
-          searchParams: {
-            saksnummer: fagsak.saksnummer,
-          },
-        })
-        .json<Dokument[]>(),
+    queryKey: [BehandlingRel.INNSYN_DOKUMENTER, behandling.uuid, behandling.versjon],
```

Comment:
hva gjør disse queryparameterene?

Reply:
Desse har ingenting med sjølve kallet å gjera. Dette er index'en til dette kallet i cachen. Så når til dømes versjon endrar seg så blir kallet gjort på nytt. Derfor vil ein gjera mindre kall om ein hadde nytta saksnr i staden for uuid og versjon